packages/typescript-estree/src/createParserServices.ts

```diff
@@ -8,16 +8,25 @@ export function createParserServices(
   program: ts.Program | null,
 ): ParserServices {
   if (!program) {
-    // we always return the node maps because
-    // (a) they don't require type info and
-    // (b) they can be useful when using some of TS's internal non-type-aware AST utils
-    return { program, ...astMaps };
+    return {
+      program,
+      emitDecoratorMetadata: undefined,
+      experimentalDecorators: undefined,
```

Comment:
[Style] Is there a reason to not let these be `?:` optional? Just consistency of whether they exist?

Two cents: I'd personally prefer to keep them out of the objects & snapshots for the sake of keeping things lean. They're not used by most users, so it's a fair bit of extra noise.

Reply:
The reason I didn't was yeah so that things were consistent - they either exist or they don't. We ran into problems in the past with properties maybe existing and it complicating things.

Eg if someone uses a different parser or an older parser and the properties don't exist - is that exactly the same as them existing but being `undefined`?
By making them exist it's much easier should we need to check down the line.